libavcodec/hw_base_encode.c

```diff
@@ -528,4 +528,4 @@ int ff_hw_base_encode_receive_packet(AVCodecContext *avctx, AVPacket *pkt)
                "size %d bytes.\n", pkt->pts, pkt->dts, pkt->size);
 
     return 0;
-}
\ No newline at end of file
+}
```

Comment:
deleted the newline ?

Reply:
We need a new line at the end right?  I added a new line.

Reply:
fixed.